.github/workflows/package_pypi.yml

```diff
@@ -38,6 +43,15 @@ jobs:
       with:
         fetch-depth: 0
 
+    # cibuildwheel only supports arm64 Linux wheels through emulation.
+    # It works fine, but it is slow. Cross-compilation is not supported yet,
```

Comment:
How slow is slow? I think for the nightlies arm64 with qemu is 4x slower.

Reply:
It takes 4h 15m whereas the build for x86_64 takes 21m. Since this is only run at the end of the day or for releases, maybe we can live with it. GitHub started offering [arm-based hosted runners](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/) in private beta on October 2023, so the expectation is that arm-based runners will be generally available some time in the near future.